pandas/io/parsers.py

```diff
@@ -716,7 +720,8 @@ def parser_f(filepath_or_buffer,
 
 
 @Appender(_read_fwf_doc)
-def read_fwf(filepath_or_buffer, colspecs='infer', widths=None, **kwds):
+def read_fwf(filepath_or_buffer, colspecs='infer', infer_nrows=100,
```

Comment:
Should add the new keyword to the end of positional arguments so as not to interfere with existing user code

Reply:
Thanks, @WillAyd.